hack/verify_boilerplate.py

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env python
```

Comment:
Why is this replacing the go one?

Reply:
Discussed in #23. The go one has a bug. Instead of fixing the bug, we are trying to use what capa is using.